backend/LexBoxApi/Auth/AuthKernel.cs

```diff
@@ -136,6 +136,17 @@ public static class AuthKernel
                         return Task.CompletedTask;
                     }
                 };
+            }).AddGoogle(googleOptions =>
+            {
+                var googleConfig = configuration.GetSection("Authentication:Google").Get<GoogleOptions>();
+                // ArgumentNullException.ThrowIfNull(googleOptions); // Eventually we'll throw if google config not found
+                if (googleConfig is not null)
+                {
+                    googleOptions.ClientId = googleConfig.ClientId;
+                    googleOptions.ClientSecret = googleConfig.ClientSecret;
+                }
+                googleOptions.AuthorizationEndpoint = "/api/login/signin-google";
```

Comment:
this is actually instructing dotnet how to contact google. That said none of this `AddGoogle` code is doing anything right now so it's not an issue. This sets up the flow where everything is handled server side meaning we don't need the JS library.

Reply:
I'll probably end up using this if I implement the "Log in with Google" button myself as mentioned in https://github.com/sillsdev/languageforge-lexbox/pull/597#issuecomment-1956221374.

Reply:
This is now being used, as my most recent push switches to backend-based flow.